client/my-sites/customer-home/cards/features/support/index.jsx

```diff
@@ -23,22 +23,18 @@ import { getSelectedEditor } from 'state/selectors/get-selected-editor';
  */
 import './style.scss';
 
-/**
- * Image dependencies
- */
-import happinessIllustration from 'assets/images/customer-home/happiness.png';
-
 const Support = ( { trackContactAction, trackDocsAction } ) => {
 	const translate = useTranslate();
 
 	return (
 		<Card className="support">
-			<CardHeading>{ translate( 'Support' ) }</CardHeading>
+			<CardHeading tagName="h2" size={ 16 }>
+				{ translate( 'Support' ) }
+			</CardHeading>
 			<h6 className="support__header customer-home__card-subheader">
-				{ translate( 'Get all the help you need' ) }
+				{ translate( 'Get all the help you need.' ) }
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.